doc/glossary.rst

```diff
@@ -213,6 +213,18 @@
                 Ceph cluster. See :ref:`the "Cluster Map" section of the
                 Architecture document<architecture_cluster_map>` for details.
 
+        Crimson
+                A next-generation OSD architecture whose core aim is the
```

Comment:
@zdover23 Unfinished business:  the first "core" here is awkward given the conflicting use later in the sentence.  Suggest s/core aim/aim/ for a future PR.

Reply:
Good point.